frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/create.ts

```diff
@@ -126,7 +126,6 @@ export const createVM = async (params: CreateVMParams) => {
     // ProcessedTemplates endpoint will reject the request if user cannot post to the namespace
     // common-templates are stored in openshift namespace, default user can read but cannot post
     template
-      .setNamespace(namespace)
```

Comment:
are you sure the user has the right to create the template in the openshift namespace? Hence the comment above.

Reply:
We are only concerned about the vm pointer to the parent template right? It should probably be enough to restore the old namespace just before this line https://github.com/openshift/console/pull/4701/files#diff-e76664ae71de5c525cc5ed6e2f799809R148

Reply:
> are you sure the user has the right to create the template in the openshift namespace? Hence the comment above.

I'm the admin on my cluster so I don't know ... ( I can, but i'm the admin ... )

@rhrazdil  @suomiy  do you know how to create a user with less previliges ?

Reply:
> are you sure the user has the right to create the template in the openshift namespace?

@suomiy Correct !!
tested with unprivileged user, can't process a template in "openshift" namespace.

> It should probably be enough to restore the old namespace

Will try this, thanks :+1: